cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationsOperations.java

```diff
@@ -97,8 +97,13 @@ public void create(String appName, ApplicationBean bean, boolean shouldStart) th
         }
 
         try {
+            Log.debug("Create app:", appName,
+                    "\n bean of the app:", bean,
```

Comment:
Please avoid newline characters in log messages; one line = one log message. Should be logged as three messages, I guess.

Reply:
Done.